cmd/metal-api/internal/service/firewall-service.go

```diff
@@ -107,6 +108,7 @@ func (r firewallResource) webService() *restful.WebService {
 		Operation("allocateFirewall").
 		Doc("allocate a firewall").
 		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Param(ws.QueryParameter("try", "try allocation before actually doing so to get informed early about possible incompatible allocation parameters").DataType("bool").DefaultValue("false")).
```

Comment:
Again thinking about this parameter I've come to the conclusion that it's a bit clunky because testing the size image constraint now requires a lot of fields from the caller (see cloud-api trying to imitate future GEPM allocation requests).

Maybe it's better to add an individual `try` endpoint to the imagesizeconstraint service, which only requires a size id and an image id. The validation code called by this endpoint can be shared with and also be called from `allocateMachine`.